ansible/roles/vault-raft-backup-agent/tasks/main.yml

```diff
@@ -1,5 +1,11 @@
 ---
 
+- name: ensure Vault snapshot config dir
+  file:
+    path: '{{ vault_snapshot_config_dir }}'
+    state: directory
+    mode: '0700'
```

Comment:
Yes, since we also have this user in the Systemd unit, this would be better. Otherwise, we cannot access it as the "vault" user:

```
# sudo -u vault vault agent -config=/etc/vault.d/vault_snapshot_agent.hcl
Error loading configuration from /etc/vault.d/vault_snapshot_agent.hcl: stat /etc/vault.d/vault_snapshot_agent.hcl: permission denied
```

Reply:
fixed in https://github.com/adfinis-sygroup/vault-raft-backup-agent/pull/10/commits/fb238976, please don't make me add Selinux contexts ;-)